.github/CODEOWNERS

```diff
@@ -0,0 +1,5 @@
+# This is a comment.
+# Each line is a file pattern followed by one or more owners.
+# See https://help.github.com/en/articles/about-code-owners#example-of-a-codeowners-file for more details and examples
+
+*       @robwalch
```

Comment:
never has there been such a true line of code

Reply:
I didn't want to subject anyone else to the same without discussing first, but feel free to add to it.